# goal
float64 depth
float64 stable_threshold
uint32 stable_periods
---
# result
float64 error
---
# feedback
float64 error
